### PR TITLE
Rename save as to duplicate #738

### LINKED ISSUE #738: Rename Save as to Duplicate
Rename save as to Duplicate for catalogue category, catalogue items, systems and manufacturers

### PR DESCRIPTION
**Pull request by asuresh-code**
## Description

Renamed the "Save as" function to "Duplicate" in Systems, Catalogue categories, Catalogue items, and Items (Function does not exist for Manufacturers).

## Testing instructions

Add a set up instructions describing how the reviewer should test the code

- [ ] Review code
- [ ] Check Actions build
- [ ] Review changes to test coverage
- [ ] {more steps here}

## Agile board tracking

connect to #738 
closes #738

### CODE REVIEW

**Comment by asuresh-code** [comment]
For the image, it is still using the SaveAs icon from the [MUI icons](https://mui.com/material-ui/material-icons/?query=saveas).

One of [these icons](https://mui.com/material-ui/material-icons/?query=duplicate) may be more appropriate now that it's renamed to duplicate.

**Review by joshuadkitenge — CHANGES_REQUESTED** [review]
Works well, Just one minor comment. 

There are still some place where "save as" is use in test names and comments etc. Can these be changed to "duplicate".

File: src/catalogue/category/catalogueCard.component.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import DeleteIcon from '@mui/icons-material/Delete';
 import EditIcon from '@mui/icons-material/Edit';
 import MoreHorizIcon from '@mui/icons-material/MoreHoriz';
-import SaveAsIcon from '@mui/icons-material/SaveAs';
+import DuplicateIcon from '@mui/icons-material/SaveAs';
```
Comment (joelvdavies) [inline src/catalogue/category/catalogueCard.component.tsx:4]: Technically this is fine as its a default export, but I would prefer it matches what MUI icons gives you when you look them up, we use their naming everywhere even if it isn't used as a save as icon here.

Comment (asuresh-code) [reply src/catalogue/category/catalogueCard.component.tsx]: Noted, just pushed a commit undoing those changes.

**Review by joelvdavies — APPROVED** [review]
Feel free to merge it now, normally we would also delete the branch using the button that appears after the merge.

**Pull request merged**